sspl_test/framework/utils/encryptor.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
```

Comment:
Can we use `framework.utils.encryptor`? If encryption logic changes in future, we need to change here again.

Reply:
agree, we need to reuse sspl code here

Reply:
If we use `framework.utils.encryptor` inside SSPL Test, it leads to an import error, hence created this new file. If we want to use the same file, we will need to figure out a way to copy it to sspl_test while building the rpms.

Reply:
for same reason sspl_constants file copy is also created i believe. We need to overcome this or we will keep replicating code & bloat so much. 
Adjusting python path should help ideally, but seems you would have tried already, may not be that straight forward.

Reply:
Yeah. Let's keep it for now.

Reply:
Alright, keep a TODO note & create a task ticket then to address in future. both for this one & sspl-constants & alike code.

Reply:
We need to avoid "do later" approach because its piling up with more TODOs. Technically we have the library is available in cortx-py-utils and can call directly instead of duplicating it.